### DISCUSSION TITLE
Nomination of @J-N-K

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-core#3501] [Core Maintainers Discussions]
Dear @openhab/core-maintainers,

I'd like to nominate @J-N-K as a core maintainer (assuming that he is willing to pick up this role).
Even though there have been some conflicts of opinion between him and me in the past, I very much acknowledge his deep understanding of our code base and I very much appreciate [his tremendous efforts since the beginning of this year](https://github.com/openhab/openhab-core/pulls?q=is%3Apr+author%3AJ-N-K+) to improve the core code.
I trust that he will act with the best intentions for the overall openHAB community and would thus be happy to have him join the core team!

**Comment by cweitkamp** [comment]
Great. I appreciate this decision. An additional core maintainer will help us a lot.

**Comment by wborn** [comment]
I also think he will be a good addition to the team of maintainers. :+1:

**Comment by kaikreuzer** [comment]
Thanks, that was a quick vote. @J-N-K welcome to the team!

Btw, @cweitkamp and @wborn: All of you are free to do nominations at any time - so whenever you feel somebody is up for the job, don't hesitate to do a nomination yourself!

**Comment by J-N-K** [comment]
Thanks for the confidence to all of you.
